### PR TITLE
CGP-0044: Increasing Mento bucket sizes

### PR DESCRIPTION
**Pull request by martinvol**

### CODE REVIEW

File: CGPs/CGP-0044.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+## Overview
+
+This Governance intends to increase the amount of the buckets, as a percent of the total amount of the unfrozen Reserve Celo buckets. The concrete proposal is to lift the current bucket sizes to `X%`.
+
+Updating the bucket sizes means that Mento has more liquidity to provide to arbitragers, as well as making exchanges suffer less slippage, which could lead to a tighter peg. 
+
+### Status
+  
+Currently, the parameter `reserveFraction` is set to 0.5%. This paramter was sufficent in the past, but as demand for Celo Stable assets growing in the last months, this parameter could be updated to be able to provide greater liquidity.
```
Comment (YazzyYaz) [inline CGPs/CGP-0044.md:20]: typo `paramter` -> `parameter`
grammar: `assets growing` -> `assets is growing`
`last months` -> `last few months`

File: CGPs/CGP-0044.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Risks
+
+* Higher bucket size means more funds at risk for the Reserve
+* Granda Mento just launched, and it could potentially mean the extra liquidity proposed here is not actually needed for the time being.
+
+## Useful Links
+
+* [Mento Doc](shttps://docs.celo.org/celo-codebase/protocol/stability/doto)
+*
```
Comment (YazzyYaz) [inline CGPs/CGP-0044.md:48]: please delete extra bullet point.

File: CGPs/CGP-0044.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+cgp: 44
+title: Increasing Mento bucket sizes
+date-created: 2021-10-12
+author: @martinvol, @rcroessmann
+status: DRAFT
+discussions-to:
```
[the remark this answers is not in the document]
Comment (martinvol) [reply CGPs/CGP-0044.md:7]:
```suggestion
discussions-to: https://forum.celo.org/t/economic-parameters-bucket-sizes-and-reserve-allocation-cgp-40-cgp-43/1997
```

File: CGPs/CGP-0044.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+author: @martinvol, @rcroessmann
+status: DRAFT
+discussions-to: 
+governance-proposal-id: [if submitted]
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+
+---
+## Overview
+
+This Governance intends to increase the amount of the buckets, as a percent of the total amount of the unfrozen Reserve Celo buckets. The concrete proposal is to lift the current bucket sizes to `X%`.
```
[the remark this answers is not in the document]
Comment (martinvol) [reply CGPs/CGP-0044.md:14]:
```suggestion
This Governance intends to increase the amount of the buckets, as a percent of the total amount of the unfrozen Reserve Celo buckets. The concrete proposal is to lift the current bucket sizes to `X%` of the total unfrozen CELO in the Reserve.
```

File: CGPs/CGP-0044.md (hunk 2 above)
Comment (martinvol) [reply CGPs/CGP-0044.md:48]:
```suggestion
* [CELo Stability Whitepaper](https://celo.org/papers/stability)
```

**Pull request merged**